packages/nc-gui/pages/account/index/[page].vue

```diff
@@ -0,0 +1,6 @@
+<template>
+  <AccountUserManagement v-if="$route.params.page === 'users'" />
+  <AccountToken v-else-if="$route.params.page === 'tokens'" />
+  <AccountAppStore v-else-if="$route.params.page === 'apps'" />
```

Comment:
Two reasons why I stick with dynamic pages,

- In the left sidebar we need to toggle the active menu item with path params it seems easier.
- In the `nestedPage` we have some custom logic based on the role it will fall back to the password reset page which seems easier this way.